src/ontology/mondo-edit.obo

```diff
@@ -377623,7 +377623,7 @@ xref: ICD10CM:C94.2 {source="Orphanet:518", source="DOID:8761", source="Orphanet
 xref: ICD10CM:C94.20 {source="DOID:8761"}
 xref: ICD9:207.2 {source="DOID:8761"}
 xref: ICDO:9910/3 {source="NCIT:C3170"}
-xref: MedDRA:C0023462 {source="EFO:0003025"}
+xref: MedDRA:10060556 {source="EFO:0003025"}
```

Comment:
@nicolevasilevsky should we also add the orcid of the person who reported this as a source?

Reply:
I was wondering the same thing, I like this idea, will do